ocs_ci/ocs/resources/pod.py

```diff
@@ -702,10 +702,24 @@ def get_ceph_tools_pod(skip_creating_pod=False):
         ToolBoxNotFoundException: In case of tool box not found
 
     """
+    if (
+        config.multicluster
+        and config.ENV_DATA.get("platform", "").lower() == constants.FUSIONAAS_PLATFORM
+        and config.ENV_DATA.get("cluster_type", "").lower()
+        == constants.MS_CONSUMER_TYPE
+    ):
+        provider_kubeconfig = os.path.join(
+            config.clusters[config.get_provider_index()].ENV_DATA["cluster_path"],
+            config.clusters[config.get_provider_index()].RUN.get("kubeconfig_location"),
+        )
+        cluster_kubeconfig = provider_kubeconfig
+    else:
+        cluster_kubeconfig = ""
     ocp_pod_obj = OCP(
         kind=constants.POD,
         namespace=config.ENV_DATA["cluster_namespace"],
         selector=constants.TOOL_APP_LABEL,
+        cluster_kubeconfig=cluster_kubeconfig,
```

Comment:
Yes because empty string the default value https://github.com/red-hat-storage/ocs-ci/blob/5f4a990659972280a65f86d9065e3c75c2962dee/ocs_ci/ocs/ocp.py#L52

Reply:
Ah, ok, sorry.